compiler/src/iree/compiler/Codegen/LLVMGPU/test/ukernel_pipeline_transform.mlir

```diff
@@ -4,7 +4,7 @@
   #hal.pipeline.binding<storage_buffer>,
   #hal.pipeline.binding<storage_buffer>
 ]>
-#executable_target_rocm_hsaco_fb = #hal.executable.target<"rocm", "rocm-hsaco-fb", {ukernels = "argmax"}>
+#executable_target_rocm_hsaco_fb = #hal.executable.target<"rocm", "rocm-hsaco-fb", {abi = "hip", iree.gpu.target = #iree_gpu.target<arch = "gfx1100", features = "", wgp = <compute =  fp64|fp32|fp16|int64|int32|int16|int8, storage =  b64|b32|b16|b8, subgroup =  shuffle|arithmetic, dot =  dp4xi8toi32, mma = [<WMMA_F32_16x16x16_F16>, <WMMA_F16_16x16x16_F16>, <WMMA_I32_16x16x16_I8>, <WMMA_I32_16x16x16_I8>, <WMMA_I32_16x16x16_I8>], subgroup_size_choices = [32, 64], max_workgroup_sizes = [1024, 1024, 1024], max_thread_count_per_workgroup = 1024, max_workgroup_memory_bytes = 65536, max_workgroup_counts = [2147483647, 2147483647, 2147483647], max_load_instruction_bits = 128, simds_per_wgp = 4, vgpr_space_bits = 8192>>, ukernels = "argmax"}>
```

Comment:
we really want to avoid doing this - please remove everything you absolutely do not need (any time these attributes change every single test using them has to be updated, and as test readability goes this makes it impossible to know what the code is actually using from this information - is the code reliant on the max workgroup count? the simds per wgp? which dot product instructions are optimal, etc? maybe it is, but when all of the info is included it's not possible to tell)

Reply:
Good catch. It turns out that I don't need to specify `#iree_gpu.target` at all since we're passing ` --iree-gpu-test-target=` on the command line. I think I mistakenly believed that I needed to specify it when I added `ukernels = "argmax"`.